helm/values-prod.yaml

```diff
@@ -2,11 +2,11 @@ image:
   version: '20230602.1121'
 resources:
   limits:
-    cpu: 4
+    cpu: 2
     memory: 7Gi
   requests:
-    cpu: 2
-    memory: 6Gi
+    cpu: 0.5
+    memory: 4Gi
```

Comment:
mem usage is very spiky, but rarely goes over 4GiB, and hasn't gone over 6GiB in the last 30d